src/appdistribution/client.ts

```diff
@@ -260,4 +306,41 @@ export class AppDistributionClient {
 
     utils.logSuccess(`Testers removed from group successfully`);
   }
+
+  async createReleaseTest(
+    releaseName: string,
+    devices: TestDevice[],
+    loginCredential?: LoginCredential,
+  ): Promise<ReleaseTest> {
+    try {
+      const response = await this.appDistroV1AlphaClient.request<ReleaseTest, ReleaseTest>({
+        method: "POST",
+        path: `${releaseName}/tests`,
+        body: {
+          deviceExecutions: devices.map((d) => this.mapDeviceToExecution(d)),
+          loginCredential,
+        },
+      });
+      utils.logSuccess(`Release test created successfully`);
```

Comment:
Nit: I see we do it throughout this file, but it is better separation of concerns to do this logging at the call site instead of inside this client file.

Reply:
Sounds good. I'll at least fix this new case just to avoid propagating the problem.